tutorials/krusty/KRUSTY/openmc.i

```diff
@@ -0,0 +1,151 @@
+################################################################################
+## KRUSTY                                                                     ##
+## Heat Pipe Microreactor Steady State                                        ##
+##                                                                            ##
+## Mahmoud (3/25) (openmc.i/solid.i/model03.py/krusty_scale.e)                ##
+################################################################################
+
+power = 4000           # (W)
+
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = krusty_scale.e
+  []
+  # [scale]
+  #  type = TransformGenerator
+  #  transform = SCALE
+  #  vector_value = '0.01 0.01 0.01'
+  #  input = fmg
+  # []
+[]
+
+[AuxVariables]
+  [cell_temperature]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [cell_in]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [material_id]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+[]
+
+[AuxKernels]
+  [cell_temperature]
+    type = CellTemperatureAux
+    variable = cell_temperature
+  []
+  [cell_in]
+    type = CellInstanceAux
+    variable = cell_in
+  []
+  [material_id]
+    type = CellMaterialIDAux
+    variable = material_id
+  []
+[]
+
+[ICs]
+  [temp]
+    type = FunctionIC
+    variable = temp
+    function = temp_ic
+  []
+[]
+
+[Functions]
+  [temp_ic]
+    type = ConstantFunction
+    value = 1073
+  []
+[]
+
+# [Problem]
+#  type = FEProblem
+#  solve = false
+# []
+ 
+[Problem]
+  type = OpenMCCellAverageProblem
+  power = ${power}
+  scaling = 100.0
+  output = 'unrelaxed_tally_std_dev'
+  # temperature_blocks = 'cavity_center clamp core gap_clamp gap_ref gap_sleeve gap_vaccan mli ref_bottom ref_top sleeve vacuum_can'
+  temperature_blocks = '12 5 14 4 11 9 6 7 13 15 10 8'
```

Comment:
can we use subdomain names instead of numbers? I find this is a bit more intuitive